code/modules/power/breaker_box.dm

```diff
@@ -20,9 +20,9 @@
 	var/update_locked = 0
 
 /obj/machinery/power/breakerbox/Destroy()
-	..()
 	for(var/datum/nano_module/rcon/R in world)
 		R.FindDevices()
```

Comment:
Please just kill this. It autoupdates and rebuilds the list on autoupdate anyway.

Reply:
Any reason not to kill the SMES one as well?

Reply:
Kill that too, I feel.